### ISSUE TITLE
3.3V rail at ~0.5V

### ISSUE THREAD

**Issue by fmckeogh** [chocol4te/mremote#28]
When powered via a battery, the 3.3V is at ~0.5V, so very likely to be an issue with the onboard regulator circuit. Needs to be fixed by OxidizeConf as demo requires battery power.

Temporary fix is to use an external known-good regulator to bypass the onboard one.

**Comment by fmckeogh** [comment]
The issue is that I failed to read the data sheet properly; `SHUTDOWN` on the TC1014 cannot be left floating. A new PCB revision should address this with a high value resistor between `SHUTDOWN` and `GND`, and then expose `SHUTDOWN` and `BATT_POSITIVE` on the header pins for an on/off switch.

**Comment by jonas-schievink** [comment]
> a high value resistor between `SHUTDOWN` and `GND`

Did you mean `SHUTDOWN` and `VCC` here?

**Comment by fmckeogh** [comment]
No, I think the the board should be normally off when a battery is connected, and shorting the two header pins with a switch should bring it on.

**Comment by fmckeogh** [comment]
Removed the OxidizeConf milestone as I have patched the board I will be using and there is no time for a new PCB run.
![IMG_0045](https://user-images.githubusercontent.com/8290136/56352959-9e64aa80-61d0-11e9-89a5-12e188c022a3.jpeg)
